src/FSharp.Formatting.ApiDocs/GenerateHtml.fs

```diff
@@ -707,7 +707,7 @@ type HtmlRender(model: ApiDocModel, ?menuTemplateFolder: string) =
             printfn "  Generating %s" outFile
             SimpleTemplating.UseFileAsSimpleTemplate(substitutions, templateOpt, outFile)
 
-        for info in model.EntityInfos do
+        for info in model.EntityInfos |> List.filter (fun info -> not info.Entity.Exclude) do
```

Comment:
Should we consider having the filtering inside `.EntityInfos` instead?
Based on the usage of this member I think this makes sense.

Reply:
I will take a look at EntifyInfos. My preference was to do it in the logic that calls `checkAccess`, but I believe that's before we've processed the comments.